plugins/for_each_line.cc

```diff
@@ -238,17 +238,17 @@ forEachLineStr(const std::string & filename,
 /*****************************************************************************/
 
 void forEachLineBlock(std::istream & stream,
-                      int64_t lineOffset,
```

Comment:
We shouldn't need to modify the interface here.  The bool which is returned from the function can be used to skip.